extra/patterns/diamods.js

```diff
@@ -0,0 +1,27 @@
+function diamods(n)
```

Comment:
The functions name should be a triangle for this program. I think you had switch name of the function for both the programs